kubernetes/reporter/pom.xml

```diff
@@ -61,4 +61,18 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>1.8</source>
+          <target>1.8</target>
```

Comment:
missed it as Jboss has lower version(19) in parent which is using java 7. now updating it 21